solr/solrj/src/java/org/apache/solr/client/solrj/request/beans/AddReplicaPayload.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.solr.client.solrj.request.beans;
+
+import org.apache.solr.common.annotation.JsonProperty;
+import org.apache.solr.common.util.ReflectMapWriter;
+
+import java.util.List;
+import java.util.Map;
+
+public class AddReplicaPayload implements ReflectMapWriter {
+  @JsonProperty
+  public String shard;
+
+  @JsonProperty
+  public String _route_;
+
+  // TODO Could 'node' be replaced (in v2 and v1) by just specifying a 'createNodeSet' of size 1?
```

Comment:
I just read the "tip" in the doc that warns you you have to pick one or the other, this would be a nice simplifying thing.   We have too many gotchas in the behaviors as is!

Reply:
OK, thanks for the +1 on the idea.  I'll delegate it to a separate ticket, since changing the API brings in 8x deprecation and other concerns. (See SOLR-15542)